src/components/Patient/PatientInfoCard.tsx

```diff
@@ -458,44 +459,48 @@ export default function PatientInfoCard(props: PatientInfoCardProps) {
                     />
                   </Badge>
                 </div>
-              </div>
-            </div>
-          </div>
-        </div>
-        <div
-          className="flex flex-col mt-4 items-center justify-end gap-4 px-4 py-1 2xl:flex-row"
-          id="consultation-buttons"
-        >
-          <PLUGIN_Component
-            __name="PatientInfoCardQuickActions"
-            encounter={encounter}
-            className="w-full lg:w-auto bg-primary-700 text-white hover:bg-primary-600"
-          />
-          {!disableButtons && (
-            <div
-              className="flex w-full flex-col gap-3 lg:w-auto 2xl:flex-row"
-              data-cy="update-encounter-button"
-            >
-              <DropdownMenu>
-                <DropdownMenuTrigger asChild>
-                  <Button variant="primary">
-                    {inactiveEncounterStatus.includes(encounter.status) ||
-                    !facilityIdExists
-                      ? t("actions")
-                      : t("update")}
-                    <ChevronDown className="ml-2 size-4" />
-                  </Button>
-                </DropdownMenuTrigger>
-                <DropdownMenuContent align="end">
-                  <EncounterActions encounter={encounter} layout="dropdown" />
+                <div
+                  className="flex flex-col items-center justify-end gap-4 px-4 py-1 2xl:flex-row"
+                  id="consultation-buttons"
+                >
                   <PLUGIN_Component
-                    __name="PatientInfoCardActions"
+                    __name="PatientInfoCardQuickActions"
                     encounter={encounter}
+                    className="w-full lg:w-auto bg-primary-700 text-white hover:bg-primary-600"
                   />
-                </DropdownMenuContent>
-              </DropdownMenu>
+                  {!disableButtons && (
+                    <div
+                      className="flex w-full flex-col gap-3 sm:w-auto"
+                      data-cy="update-encounter-button"
+                    >
+                      <DropdownMenu>
+                        <DropdownMenuTrigger asChild>
+                          <Button variant="primary">
+                            {inactiveEncounterStatus.includes(
+                              encounter.status,
+                            ) || !facilityIdExists
+                              ? t("actions")
+                              : t("update")}
+                            <ChevronDown className="ml-2 size-4" />
+                          </Button>
+                        </DropdownMenuTrigger>
+                        <DropdownMenuContent className="w-(--radix-dropdown-menu-trigger-width) sm:w-auto">
```

Comment:
let's preserve the align property (it's center by default, which works for smaller screens, but not for larger ones). So adjust accordingly - end for bigger and center for smaller screens.